draft-scenarios/clash/gold_rush.tex

```diff
@@ -0,0 +1,164 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Solo/Clash/Alliance Scenario}{Gold Rush}{\images/gold-mine.png}
+
+\begin{multicols}{2}
+
+\textbf{Author:} farmaazon
+
+\textbf{Source:} \href{https://discordapp.com/channels/740870068178649108/1344400556717768865/1344400556717768865}{Archon Studios Discord}
+
+\textit{Your liege, engaged in costly wars in distant lands, has tasked you with extracting as much wealth as possible from this region. More importantly, you must find the legendary Grail to secure his ultimate victory.}
+
+\subsection*{\MakeUppercase{Scenario Length}}
+
+This Scenario is played over 13 Rounds.
+
+\subsection*{\MakeUppercase{Player Setup}}
+
+\textbf{Player Count:} 1, 2 or 4 (Alliance variant)
+
+\textbf{Starting Resources:} 15 \svg{gold}, 6 \svg{building_materials}, 1 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 0 \svg{building_materials}, 0 \svg{valuables}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item A Pack of \svgunit{bronze} Units with the hightst Recruitment cost
+  \item A Few \svgunit{silver} Units with the lowest Recruitment cost
+\end{itemize}
+
+\textbf{Town Buildings:} \svgunit{bronze} Dwelling
+
+\textbf{Map Tile Pool:} Each player takes 3 Far (II--III) Map Tiles with at least one Settlement.
+
+\subsection*{\MakeUppercase{Map Setup}}
+
+Take the following Map Tiles and arrange them as shown in the Scenario map layout ($P$ stands for the number of players):
+\begin{itemize}
+  \item P × Starting (I) Map Tile
+  \item 1 × Near (IV--V) Map Tile (N1), plus:
+  \begin{itemize}
+    \item \textbf{For Solo Play:} 1 × additional Near (IV--V) Map Tile containing an Obelisk
+    \item \textbf{For PvP Play:} 2P × additional Near (IV--V) Map Tiles, with exactly $P$ of them containing an Obelisk; Tiles with Obelisks cannot be adjacent to the N1 Tile
+  \end{itemize}
+  \item 2 × Center (VI--VII) Map Tile (C2 and \#C1\footnote{If you do not have the Tower expansion, you may use C1 instead of \#C1, with two changes: the \textit{Dragon Utopia} is treated as a Settlement, and the \textit{Warrior's Tomb} is protected by Neutral Units instead of the Shrine. All special rules related to \#C1 should then be applied to C1.})
+\end{itemize}
+\subsection*{\MakeUppercase{Preparation}}
+
+Remove the following Cards from their respective Decks and organize them into 5 separate piles\footnote{If playing as Lord Haart, replace his starting \textit{Estates} Ability with \textit{Scouting}.}:
+  
+\begin{enumerate}
+  \item \textit{Estates} Ability Card
+  \item Resource-generating Spell Cards (\textit{View Airs} and any similar Spells)
+  \item All \textit{Statue of Legion} Artifact pieces (\textit{X of Legion} Artifacts)
+  \item \textit{Obelisk Pile}: remaining non-Relic Artifacts that grant resources (including those with one of the options) and have no ongoing effect
```

Comment:
```suggestion
  \item \textit{Obelisk Pile}: remaining non-Relic Artifacts that grant resources (including those with one of the options) and don't have \svg{ongoing} effect
```
The visual clue will make those cards easier to identify. Please use this glyph from the rewrite: https://github.com/Heegu-sama/Homm3BG/blob/main/assets/glyphs/ongoing.svg